macos/Sources/Features/Command Palette/CommandPalette.swift

```diff
@@ -44,6 +44,12 @@ struct CommandPaletteView: View {
     }
 
     var body: some View {
+        @State var scheme: ColorScheme = if OSColor(backgroundColor).isLightColor {
```

Comment:
I'm pretty sure this doesn't need to be `@State` or `var` (can just be a plain `let`). I tested that locally and it worked fine is there something I'm missing?